sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/TopicDescription.java

```diff
@@ -657,7 +736,7 @@ public Boolean isEnableExpress() {
      * @param enableExpress the enableExpress value to set.
      * @return the TopicDescription object itself.
      */
-    public TopicDescription setEnableExpress(Boolean enableExpress) {
+    TopicDescription setEnableExpress(Boolean enableExpress) {
         this.enableExpress = enableExpress;
```

Comment:
setEnableExpress -> enableExpress()

Reply:
This is package-private, so it doesn't matter.